packages/react/src/components/Tabs/Tabs.js

```diff
@@ -116,6 +117,12 @@ export default class Tabs extends React.Component {
     return React.Children.map(this.props.children, tab => tab);
   }
 
+  getEnabledTabs = () =>
+    React.Children.toArray(this.props.children).reduce(
+      (acc, tab, index) => (!tab.props.disabled ? acc.concat(index) : acc),
```

Comment:
When we're looping through children, do we ever need to do null checks? Wasn't sure about `toArray` but was just remembering some issues with UI Shell where this was coming up.

Reply:
as far as I know `React.Children.toArray` removes nulls

ref https://github.com/facebook/react/blob/master/packages/react/src/__tests__/ReactChildren-test.js#L815